### PR TITLE
[2.4] Update setup.py of monai integration folder

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
### Description

This PR updates the dependencies and version of monai_nvflare build configuration (setup.py).

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: integration/monai/setup.py [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
-    version = "0.2.6"
+    version = "0.2.7"
 else:
     package_name = "monai-nvflare-nightly"
     today = datetime.date.today().timetuple()
     year = today[0] % 1000
     month = today[1]
     day = today[2]
-    version = f"0.2.3.{year:02d}{month:02d}{day:02d}"
+    version = f"0.2.6.{year:02d}{month:02d}{day:02d}"
```
[the remark this answers is not in the document]
Comment (IsaacYangSLA) [reply integration/monai/setup.py:34]: Nightly is one version before the actual release.  And the nightly is not built/released to PyPi.  This portion is for user's own installation from source.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Pull request merged**